focus-ios/focus-ios-tests/SmokeTest.xctestplan

```diff
@@ -20,17 +20,20 @@
         "DragAndDropTest",
         "FindInPageTest\/testFindInPageURLBarElement()",
         "OpenInFocusTest\/testOpenViaSafari()",
+        "PageShortcutsTest\/testAddRemoveShortcut()",
         "PastenGoTest\/testClipboard()",
         "QuickAddAutocompleteURLTest",
         "SearchProviderTest\/testAddRemoveCustomSearchProvider()",
         "SearchProviderTest\/testDuckDuckGoSearchProvider()",
         "SearchProviderTest\/testPreventionOfRemovingDefaultSearchProvider()",
         "SearchProviderTest\/testWikipediaSearchProvider()",
         "SearchSuggestionsPromptTest",
-        "SettingAppearanceTest\/testAddRemoveCustomDomain()",
-        "SettingAppearanceTest\/testDisableAutocomplete()",
-        "SettingAppearanceTest\/testOpenInSafari()",
-        "PageShortcutsTest\/testAddRemoveShortcut()",
+        "SettingTest\/testAddRemoveCustomDomain()",
+        "SettingTest\/testAutocompleteCustomDomain()",
+        "SettingTest\/testDisableAutocomplete()",
+        "SettingTest\/testEnableDisableAutocomplete()",
+        "SettingTest\/testOpenInSafari()",
+        "SettingTest\/testReEnableAutoComplete()",
```

Comment:
I ensure that these `SettingTest`s are the full functional tests according to TestRail.